examples/charm++/load_balancing/stencil3d/stencil3d.C

```diff
@@ -185,7 +185,7 @@ class Stencil: public CBase_Stencil {
       // set period arbitrarily small so that LB occurs when AtSync is called
       // this is in case the default LBPERIOD is larger than the time to complete LBPERIOD_ITER
```

Comment:
Is this still an issue with the new LB framework?

Reply:
No, it's not, I'll remove it.